docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc

```diff
@@ -0,0 +1,23 @@
+= Creating OpenShift projects
+
+Source::
+Developer
+
+Stimulus::
+Creates new OpenShift project
+
+Environment::
+Any {zone}
+
+Artifact::
+oc / OpenShift web console
+
+Response::
+Project created
+
+Response measure::
+Project (and associated namespace) belongs to the user's default organization
+
+Rationale::
+To enable an OpenShift native experience.
+Developers should be able to create projects in the OpenShift web console or with `oc new-project bar` as if they're on their own OpenShift cluster.
```

Comment:
```suggestion
Namespace ownership must not be tied to individuals but to the organization.
This prevents that team mates get locked out from a namespace should its creator forgett to update its permission.
We consider, giving all members access by default, a safer default.
```

Reply:
We need to cover both. We can do them separately or mixed. I am uncertain which is better, but tend towards separate requirements. So I guess this one can be merged as is.

Reply:
I'll add another page for the organization-level RBAC requirement.